ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource

```diff
@@ -26,7 +26,6 @@ Open Data Science Projects Home Page
     SeleniumLibrary.Click Link      Data Science Projects
     Wait for RHODS Dashboard to Load    wait_for_cards=${FALSE}    expected_page=Data science projects
     SeleniumLibrary.Wait Until Page Contains    View your existing projects or create new projects.    timeout=30
-    Maybe Wait For Dashboard Loading Spinner Page
```

Comment:
@bdattoma is this correct?

Reply:
hm I'm not 100% sure, I would like to see it in multiple runs